robot/impl/module_lifecycle_test.go

```diff
@@ -420,10 +420,10 @@ func TestCrashedModuleDependentRecovery(t *testing.T) {
 	})
 
 	// Test that the crashed module is added to failedModules.
-	// Reconfigure will preserve modules still in the config (UpdateFailedModules only clears
+	// Reconfigure will preserve modules still in the config (ClearFailedModules only clears
```

Comment:
ClearFailedModules clears all modules from failed modules, right? we should just remove these comments